### PR TITLE
Add unit tests for comment service

### PR DESCRIPTION
**Pull request by andrewyguo**
## Notion ticket link

[Ticket ](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=d9efc1595f954086b7ece906ccdd62c4)

## Implementation description

- Add unit tests for story service
- Remove `update_comments` mutation in `comment_service`

## Steps to test

1. Read through unit tests, check that they make sense
2. Make sure nothing is broken after removing  `update_comments` mutation in `comment_service`

## What should reviewers focus on?

-

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/services/implementations/comment_service.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def update_comment(self, updated_comment, user_id):
             for key, value in updated_comment.__dict__.items():
                 setattr(comment, key, value)
 
-            if updated_comment["resolved"] == True:
+            if updated_comment.resolved == True:
```
Comment (andrewyguo) [inline backend/python/app/services/implementations/comment_service.py:137]: blame @gaoxk

File: backend/python/app/tests/services/test_comment_service.py [hunk 2]
```diff
@@ -1,24 +1,105 @@
...
 
+    assert resp == comment_obj
+    assert comment_obj.comment_index == 0
 
-def test_update_comment_invalid_id():
-    pass
+    # Multiple comments
+    comment_2 = CreateCommentDTO(
+        story_translation_content_id=story_translation_contents[0]["id"],
+        content="Orwellian State-Mandated Unit Tests",
```
Comment (gaoxk) [inline backend/python/app/tests/services/test_comment_service.py:39]: 🥴

File: backend/python/app/tests/services/test_comment_service.py [hunk 3]
```diff
@@ -1,24 +1,105 @@
...
+    comment = CreateCommentDTO(
+        story_translation_content_id=story_translation_contents[0]["id"],
+        content="Orwellian State-Mandated Unit Tests",
+    )
+    resp = services["comment"].create_comment(comment.__dict__, translator.id)
+    comment_obj = CommentAll.query.get(resp.id)
+
+    assert comment_obj.content == "Orwellian State-Mandated Unit Tests"
+
+    comment_2 = UpdateCommentRequestDTO(resp)
```
Comment (gaoxk) [inline backend/python/app/tests/services/test_comment_service.py:69]: can you test updating the resolved col?

Comment (gaoxk) [reply backend/python/app/tests/services/test_comment_service.py:69]: resolving 1 comment should resolve it's entire thread too

**Review by gaoxk — CHANGES_REQUESTED** [review]

File: backend/python/app/tests/services/test_comment_service.py [hunk 4]
```diff
@@ -1,24 +1,146 @@
...
+    comment_2.content = "Big Brother is watching you"
+
+    resp_2 = services["comment"].update_comment(comment_2, translator.id)
+    comment_obj_2 = CommentAll.query.get(resp_2.id)
+
+    assert comment_obj_2.content == "Big Brother is watching you"
+
+
+def test_resolve_comment(app, db, services):
```
Comment (gaoxk) [inline backend/python/app/tests/services/test_comment_service.py:78]: nit: change to `test_update_comment_resolve`

File: backend/python/app/tests/helpers/comment_helpers.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class UpdateCommentRequestDTO(Mapping):
+    def __init__(self, comment):
+        self.id = comment.id
+        self.resolved = comment.resolved
+        self.content = comment.content
+        self._dict = dict(id=comment.id, content=comment.content)
+
+    def __getitem__(self, key):
```
Comment (gaoxk) [inline backend/python/app/tests/helpers/comment_helpers.py:20]: nit: copy the comment from StoryTranslationContentRequestDTO
`# These functions are necessary for bulk_update_mappings to work`

**Review by gaoxk — APPROVED** [review]
feel free to merge after addressing nits!

**Pull request merged**